x-pack/plugins/rollup/public/crud_app/services/api.js

```diff
@@ -31,7 +33,8 @@ export async function deleteJobs(jobIds) {
 
 export async function createJob(job) {
   const body = { job };
-  return await getHttp().put(`${apiPrefix}/create`, body);
+  const action = getHttp().put(`${apiPrefix}/create`, body);
```

Comment:
To me "action" is synchronous. Here it is a "request", it can succeed or fail. It would then be clearer that is a promise in the `trackUserRequest()` method